src/stores/Binding/Store.ts

```diff
@@ -532,6 +387,17 @@ const getInitialState = (
             false,
             'Binding dependent state prefilled'
         );
+
+        return {
+            addedCollections: draftedBindings
+                ? difference(
+                      draftedBindings.map((binding) =>
+                          getCollectionName(binding)
+                      ),
+                      liveBindings.map((binding) => getCollectionName(binding))
+                  )
+                : [],
```

Comment:
We don't need to change anything here... but we could _maybe_ make this a bit eaiser on processing since if we are here we have already done through at least one of these arrays. However, it could end up getting more complex to "cache" off one of these lists while supporting multiple cases.

Reply:
Actually I guess we always know we have a `drafterBindings` here... so potentially worth it to make a shared array that is built up in the loop above. But not sure how that would impact performance with the `set`